ProjectLighthouse.Servers.Website/Pages/Moderation/NewCasePage.cshtml.cs

```diff
@@ -12,14 +12,17 @@ namespace LBPUnion.ProjectLighthouse.Servers.Website.Pages.Moderation;
 public class NewCasePage : BaseLayout
 {
     public NewCasePage(DatabaseContext database) : base(database)
-    {}
+    { }
```

Comment:
```suggestion
    {}
```

Reply:
Should there be extra space here (I am oblivious to style guidelines, if i'm wrong let me know)

Reply:
Rider did this for me

Reply:
again outside the scope of this PR but we need project linter configuration so it doesn't change every rider version

Reply:
This was the styling convention far before 2023.2

Reply:
fair enough